doc/configuration.rst

```diff
@@ -1573,6 +1573,8 @@ Arguments:
     target.
   - explicit_sftp_mode (bool, default=False): if set to True, `put()` and `get()` will
     explicitly use the SFTP protocol for file transfers instead of scp's default protocol
+  - explicit_scp_mode (bool, default=False): if set to True, `put()` and `get()` will
```

Comment:
`scp()` as well. While at it, could you add that also to `explicit_sftp_mode` (implemented in #1207)?